lib/pages/Current.dart

```diff
@@ -44,7 +34,7 @@ class _CurrentState extends State<Current> {
                     widget: SizedBox(height: 10),
                   ),
                   SizedBox(height: 20),
```

Comment:
The alignment of the time/date is centered, but in the designs it seems to be oriented on the left.
<img width="294" alt="Screen Shot 2021-04-06 at 1 46 13 AM" src="https://user-images.githubusercontent.com/17262023/113664515-e0b37a80-9679-11eb-9f04-4365df785fd5.png">

Reply:
Whoops, thanks! Will fix